doc/manuals/devel/httpHeaders.md

```diff
@@ -0,0 +1,281 @@
+10) [X-Real-IP](#10-x-real-ip)
+
+11) [Access-Control-Allow-Origin](#11--access-control-allow-origin)
+12) [Access-Control-Allow-Headers](#12-access-control-allow-headers)                                                                   
+13) [Access-Control-Allow-Methods](#13-access-control-allow-methods)
+14) [Access-Control-Max-Age](#14-access-control-max-age)                                 
+15) [Access-Control-Expose-Headers](#15-access-control-expose-headers)
+16) [Allow](#16-allow)                                                                                    
+17) [Location](#17-location)                                                 
+18) [Ngsiv2-AttrsFormat](#18-ngsiv2-attrsformat)                                    
+19) [Fiware-Service](#19-fiware-service)
+20) [Fiware-Servicepath](#20-fiware-servicepath)
+21) [Fiware-Total-Count](#21-fiware-total-count)
+22) [Fiware-Correlator](#22-fiware-correlator)                                     
+
+23) [X-Auth-Token](#23-x-auth-token)
+
+
+
+# HTTP headers used by Orion with their description
+
+
+
+## 1. Accept
+
+Accept HTTP header is used in incoming HTTP requests received by Orion. 
+Accept HTTP header is used in the request to specify which MIME type the client accepts. Typically its value is `application/json` (for JSON MIME type) although some operations in the NGSIv2 API also allow `text/plain`. 
+For example in request URL if we use:
+	
+    curl ... -H 'Accept: application/json'
+		 
+On this request we'll get the response data in well defined JSON format.
+
+The API response payloads in this specification are based on `application/json` and  (for attribute value 
+type operation) `text/plain` MIME types. Clients issuing HTTP requests with accept types different 
+than those will get a `406 Not Acceptable` error.
+
+[Top](#top)
+
+## 2. Content-Length
+
+This header is used by both requests and responses.
+Content-length HTTP header is a mandatory header in Orion responses.
+It defines the length of the request and response body in bytes.
+Orion Context Broker expects always a Content-Length header in all client requests, otherwise the client will receive a `411 Length Required` response. 
+This is due to the way the underlying HTTP library (libmicrohttpd) works.
+	
+    Content-Length: 34	
+
+[Top](#top)
+ 
+## 3. Content-Type 
+
+It is used to specify the MIME type of the request or response. Typically its value is `application/json` (for JSON MIME type) although some operations in the NGSIv2 API also allow `text/plain`.
+
+    curl ... -H 'Content-Type: application/json'
+
+This request will send data in well defined JSON format.
+
+[Top](#top)
+ 
+## 4. Connection
+ 
+ This header is used by both requests and responses.
+ Orion is not currently doing anything with this header (apart from checking if its value is `close` to print a log trace).
+	
+    Connection: Keep-Alive
+    	
+[Top](#top)	
```

Comment:
After PR https://github.com/telefonicaid/fiware-orion/pull/3599 I understand that Orion is no longer processing this header in any way. If I'm right, then this section should be removed (including the corresponding adjustement in the number sequence and ToC above).

Reply:
Fixed in 512f9cd